src/main/java/main/view/Project/TestResult.java

```diff
@@ -27,51 +27,53 @@ public void doPost(HttpServletRequest req, HttpServletResponse resp){
             testResult = session.controllerFactory.getHandler(testResult).create(testResult);
             setJSONContentType(resp);
             resp.getWriter().write(mapper.serialize(testResult));
-        }catch (Exception e) {
+        } catch (Exception e) {
             handleException(resp, e);
         }
     }
 
     @Override
-    public void doGet(HttpServletRequest req, HttpServletResponse resp){
+    public void doGet(HttpServletRequest req, HttpServletResponse resp) {
         setPostResponseHeaders(resp);
         setEncoding(resp);
         try {
             Session session = createSession(req);
             TestResultDto testResultTemplate = new TestResultDto();
             testResultTemplate.getSearchTemplateFromRequestParameters(req);
             List<TestResultDto> testResults = session.controllerFactory.getHandler(testResultTemplate).get(testResultTemplate);
+            testResults.forEach(testResultDto -> testResultDto.getAttachments().
+                    forEach(testResultAttachmentDto -> testResultAttachmentDto.setPath(null)));
```

Comment:
```suggestion
                    forEach(testResultAttachmentDto -> testResultAttachmentDto.setPath(null)));
```

is it to hide path in the response?

Reply:
exactly